pkg/actions/addon/create.go

```diff
@@ -27,11 +27,11 @@ const (
 
 func (a *Manager) Create(addon *api.Addon, wait bool) error {
 	version := addon.Version
-	if addon.Version == "latest" {
+	if version == "latest" || !strings.Contains(version, "eksbuild") && version != "" {
```

Comment:
what if they just specify `1.7.5.eksbuild` and want to let us figure out if its eksbuild1 or eksbuild2?

Reply:
yep, good question.. I can change the highlighted line^ to check if the string contains `eksbuild-1` or `eksbuild-2`! I was hoping to tackle both of these by checking for `eksbuild` but you're right, and it's an easy fix!

Reply:
or we can just exclude `ekdsbuild-`, assuming there is something after the `-`. does that sound like a reasonable catch-all?

Reply:
Mhmmm I can see the value in not specifying the eksbuild number though, it's a reasonable use case

Reply:
Agreed, in that case I will just remove the bit of code that parses the version and throws the `malformed version` error. It will just return an error if no matching version was found! :)

Reply:
sounds good!